docs/core/testing/unit-testing-platform-extensions-output.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Microsoft.Testing.Platform output extensions
+description: Learn about the various Microsoft.Testing.Platform output extensions and how to use them.
+author: nohwnd
+ms.author: jajares
+ms.date: 08/26/2024
+---
+
+# Output extensions
+
+This article list and explains all `Microsoft Testing Platform` extensions related to the terminal output.
+
+## Terminal test reporter
+
+Terminal test reporter is the default implementation of status and progress reporting to the terminal (console).
+
+It comes built-in with Microsoft.Testing.Platform, and offers ANSI and non-ANSI mode, and progress indicator.
```

Comment:
```suggestion
It comes built-in with `Microsoft.Testing.Platform`, and offers ANSI and non-ANSI mode, and progress indicator.
```

Reply:
This one contradicts the style in the other one. I suggest using **Microsoft.Testing.Platform**.

Reply:
```suggestion
It comes built-in with **Microsoft.Testing.Platform**, and offers ANSI and non-ANSI mode, and progress indicator.
```

Reply:
Used bold, as suggested.